admin/tests/acceptance/authenticated/autonomous-courses/autonomous-courses-test.js

```diff
@@ -51,64 +51,84 @@ module('Acceptance | Autonomous courses | Autonomous course', function (hooks) {
       });
     });
 
-    module('details page', function () {
-      test('it should display details of the autonomous course', async function (assert) {
+    module('list page', function () {
```

Comment:
Pourquoi ne pas garder les tests d'affichage `'it should display a list of autonomous-courses'` et `'it should display a button to create a new autonomous course'` ?

Reply:
le rebase qui a fait nimp